extensions/blocks/eventbrite/style.scss

```diff
@@ -1,4 +1,13 @@
 .wp-block-jetpack-eventbrite {
+	// Make sure the widget embed container is always as tall as the
+	// default/minimum value set by Eventbrite (425px).
+	&:not( .wp-block-button ) {
+		height: auto !important;
+		iframe {
+			height: 425px;
+		}
+	}
```

Comment:
This seems fragile as Eventbrite could easily change the embed and the 425px height without any notice (but I can't think of a better way to resolve this issue, at the moment) 😕